trunk/src/constrs/solelmI.F

```diff
@@ -175,5 +177,24 @@ subroutine solelmI(Mdle, ZdofH,ZdofE,ZdofV)
 c  ...loop through local H(div) dof
       enddo
 c
+c
+      end subroutine solelmI_coms
+c
+c
+c-----------------------------------------------------------------------
+c  ...see routine solelmI_coms for description
+c     last rev: Sep 2023
+      subroutine solelmI(Mdle, ZdofH,ZdofE,ZdofV)
```

Comment:
I'd thought of doing that at first but hesitated because we've not been using optional arguments anywhere in the source code so far, and I figured introducing it in just one single place might be strange. It would also change the usual order of arguments (inputs first, outputs second) unless the user specifies explicitly which arguments are passed in which place of the input.

But I agree it's somewhat cleaner to work with optional arguments.

Function overloading and using interfaces would be another route that's potentially better.